index.html

```diff
@@ -0,0 +1,255 @@
+    <section class="menu-click">
+      <div class="close-container">
+        <img class="close-icon" src="./img/cross-icon.png" alt="close icon" />
+      </div>
+      <ul class="menu-text">
+        <li>
+          <a href="#Home" class="nav-link">Home</a>
+        </li>
+        <li>
+          <a href="#about" class="nav-link">About</a>
+        </li>
+        <li>
+          <a href="#contact-form" class="nav-link">Contact</a>
+        </li>
+      </ul>
+    </section>
+
+    <!-- page 2 -->
+    <div class="page2 about-page">
+      <!-- about -->
+      <section class="about">
+        <div class="about-titles">
+        <h3 class="sub-title">Hello! United Nation Climate Change Conference</h3>
+        <h1 class="about-title">
+          Sharm el-Sheikh Climate Change Conference - November 2022
+        </h1>
+        <p class="paragraph1 color">
+          At the Sharm el-Sheikh Climate Change Conference (COP 27), countries came together to take action towards achieving the world’s collective climate goals as agreed under the Paris Agreement and the Convention. The conference took place from 6-20 November 2022 in Sharm el-Sheikh, Egypt. Heads of State and Government attended the Sharm el-Sheikh Climate Implementation Summit on 7 and 8 November. A high-level segment primarily attended by Ministers took place from 15-18 November.
+        </p>
+        </div>
+        <h2 class="contact">
+          Please contact us per Email for any further questions<br />
+          Sharm el-Sheikh Climate Change 2022!<br />
+          <a href="#">webmaster@unfccc.int.</a>
+        </h2>
+      </section>
+
+      <!-- logo -->
+      <section class="about-logo">
+        <h2 class="">
+          Sharm el-Sheikh Climate Change Conference - November 2022
+        </h2>
+        <hr class="line1" />
+        <p class="address">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit onsectetur
+          adipisicing elit
+        </p>
+        <img
+          class="about-img"
+          src="./img/logo.png"
+          alt="2013 comferance image"
+        />
+        <hr class="line3" />
+      </section>
+      <!-- global summit -->
+      <section class="global-summit">
+        <h2 class="">
+          Sharm el-Sheikh Climate Change Conference - November 2022
+        </h2>
+        <hr class="line1" />
+        <p class="address">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit onsectetur
+          adipisicing elit
+        </p>
+        <div class="global-img">
+          <img
+            class="about-img"
+            src="./img/cop27.jpg"
+            alt="2013 comferance image"
+          />
+          <img
+            class="about-img"
+            src="./img/cop27_18_11.jpg"
+            alt="2011 comferance image"
+          />
+        </div>
+      </section>
+      <!-- footer -->
+      <footer class="footer about-footer">
+        <img class="logo-img" src="./img/logo.png" alt="logo image" />
```

Comment:
- [optional]please, this is only optional because the previous reviewer didn't mention it. Your footer section of the mobile about page is missing the partners section. Please fix that
yours
![Screenshot (759)](https://user-images.githubusercontent.com/65298048/212440198-b923410f-c936-469c-88f9-161448b6853c.png)
original design
![Screenshot (760)](https://user-images.githubusercontent.com/65298048/212440263-6355e89c-fdad-4f67-8234-e8d26cae9b91.png)